reference/configuration/framework.rst

```diff
@@ -1973,7 +2024,11 @@ adapter
 
 **type**: ``string`` **default**: ``cache.app``
 
-The name of the adapter to use. You could also use your own implementation.
+The service name of the adapter to use. You can specify one of the default
+services that follow the pattern ``cache.adapter.[type]``. Alternatively you
+can specify another cache pool as base, which will make this pool inherit the
+settings from the base pool as defaults. You can also use your own
+implementation with a service tagged as ``cache.pool`` if you need to.
```

Comment:
adding the tag means the service's constructor must have 3 arguments (`$provider, string $namespace = '', int $defaultLifetime = 0`) - if we can omit this, maybe that's better :)